azurerm/resource_arm_container_group.go

```diff
@@ -1002,6 +1084,61 @@ func flattenContainerVolumes(volumeMounts *[]containerinstance.VolumeMount, cont
 	return volumeConfigs
 }
 
+func flattenContainerProbes(input *containerinstance.ContainerProbe) []interface{} {
+	outputs := make([]interface{}, 0)
+	if input == nil {
+		return outputs
+	}
+
+	output := make(map[string]interface{})
+
+	if v := input.Exec; v != nil {
+		output["exec"] = *v.Command
+	}
+
+	if v := input.HTTPGet; v != nil {
```

Comment:
_minor_ i would change this to
```suggestion
	if get := input.HTTPGet; v != nil {
```

so we can use it below